Add installation page (#4)
s-ccs/workshop_unfold_2025: website/installation.qmd

---
title: "Installation guide"
---

> To actively partake in the workshop and make use of the notebooks provided by us you will need to install both Julia and Pluto.jl on your system. The following is a minimal guide on how to do this. You can find more extensive information in the [Pluto.jl documentation](https://plutojl.org/en/docs/install/) (of which this guide is base on), and also watch their [instruction video](https://computationalthinking.mit.edu/Fall24/installation/).

Reviewer: ```suggestion
To actively participate in the workshop and use the notebooks we provided, you will need to install both Julia and Pluto.jl on your system. The following is a minimal guide on how to do this. You can find more extensive information in the [Pluto.jl documentation](https://plutojl.org/en/docs/install/) (which this guide is based on) and also watch their [instruction video](https://computationalthinking.mit.edu/Fall24/installation/).
```

::: {.callout-note}
Please note that we will not be able to help with installing Julia and/or Pluto during the workshop. If you experience problems please contact us *before* the day of the workshop.
:::

# Step 1: Installing Julia

> In most cases, goin to [https://julialang.org/downloads](https://julialang.org/downloads) and installing the current stable release is sufficient.

Reviewer: ```suggestion
In most cases, going to [https://julialang.org/downloads](https://julialang.org/downloads) and installing the current stable release is sufficient.
```
